modules/solid_mechanics/test/tests/power_law_creep/tests

```diff
@@ -127,4 +127,69 @@
       detail = 'calculate a perfect Jacobian with the small strain formulation'
     []
   []
+  [composite_power_law_creep]
+    type = 'CSVDiff'
+    input = 'composite_power_law_creep.i'
+    csvdiff = 'composite_power_law_creep_tests_soln_0011.csv'
+    design = 'CompositePowerLawCreepStressUpdate.md'
+    requirement = "The system shall provide a phase-dependent power law creep that computes the strain with different material properties."
+    issues = '#28368'
+  []
+  [composite_power_law_creep_single_material]
+    type = 'CSVDiff'
+    input = 'composite_power_law_creep_single_material.i'
+    csvdiff = 'composite_power_law_creep_single_material_tests_soln_0011.csv'
+    design = 'CompositePowerLawCreepStressUpdate.md'
+    requirement = "The system shall provide a phase-dependent power law creep that gives the same output as power_law_creep.i if given the same materials to all the phases."
+    issues = '#28368'
+  []
+  [composite_power_law_creep_small_strain]
+    type = 'CSVDiff'
+    input = 'composite_power_law_creep_small_strain.i'
+    csvdiff = 'composite_power_law_creep_small_strain_tests_soln_0011.csv'
+    design = 'CompositePowerLawCreepStressUpdate.md'
+    requirement = "The system shall provide a phase-dependent power law creep that works with small strain."
+    issues = '#28368'
+  []
+  [composite_power_law_creep_plasticity]
+    type = 'CSVDiff'
+    input = 'composite_power_law_creep_plasticity.i'
+    csvdiff = 'composite_power_law_creep_plasticity_tests_soln_0011.csv'
+    design = 'CompositePowerLawCreepStressUpdate.md'
+    requirement = "The system shall provide a phase-dependent power law creep that works with plasticity."
+    issues = '#28368'
+  []
+  [except_composite_power_law_activation_energy]
+    type = 'RunException'
+    input = 'composite_power_law_creep.i'
+    cli_args = 'Materials/power_law_creep/activation_energy = 3.0e5  3.5e5  4.5e5'
```

Comment:
```suggestion
    cli_args = 'Materials/power_law_creep/activation_energy=3.0e5  3.5e5  4.5e5'
```
The spaces around the equal sign can cause confusion for the command line parser